javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount > 0 && Number.isInteger(amount)) {
+    var deposit = new Transaction('deposit', amount);
+    this.transactions.push(deposit);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  if (amount > 0 && Number.isInteger(amount)) {
+    var withdrawal = new Transaction('withdrawal', amount);
+    this.transactions.push(withdrawal);
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  var balance = 0;
+  if (this.transactions.length === 0) {
+    return 0;
```

Comment:
This conditional isn't necessary. If the length of the array is `0`, then the loop would never start, so you end up returning `0` anyway